src/InstrumentationEngine.WindowsAzureDiagnostics.NuGet/InstrumentationEngine.WindowsAzureDiagnostics.csproj

```diff
@@ -0,0 +1,48 @@
+﻿<?xml version="1.0" encoding="utf-8"?>
+<!-- Copyright (c) Microsoft Corporation. All rights reserved.
+     Licensed under the MIT License. -->
+<Project>
+  <PropertyGroup>
+    <AppendTargetFrameworkToOutputPath>false</AppendTargetFrameworkToOutputPath>
+    <TargetExt>.nupkg</TargetExt>
+    <PackageId>Microsoft.InstrumentationEngine.WindowsAzureDiagnostics</PackageId>
+    <CopyBuildOutputToOutputDirectory>false</CopyBuildOutputToOutputDirectory>
+    <CopyOutputSymbolsToOutputDirectory>false</CopyOutputSymbolsToOutputDirectory>
+    <GeneratePackageOnBuild>true</GeneratePackageOnBuild>
+  </PropertyGroup>
+  <Import Project="$([MSBuild]::GetDirectoryNameOfFileAbove($(MSBuildProjectDirectory), 'EnlistmentRoot.marker'))\build\Common.props" />
+  <Import Project="$(EnlistmentRoot)\build\Packaging.props" />
+  <Import Project="Sdk.props" Sdk="Microsoft.NET.Sdk" />
+  <PropertyGroup>
+    <GenerateAssemblyFileVersionAttribute>false</GenerateAssemblyFileVersionAttribute>
+    <GenerateAssemblyInfo>false</GenerateAssemblyInfo>
+    <ResolveNuGetPackages>true</ResolveNuGetPackages>
+    <TargetFramework>netstandard2.0</TargetFramework>
+    <IncludeBuildOutput>false</IncludeBuildOutput>
+    <NuspecFile>InstrumentationEngine.WindowsAzureDiagnostics.nuspec</NuspecFile>
+    <MsiSuffix Condition=" '$(Configuration)' == 'Debug' ">_$(Configuration)</MsiSuffix>
```

Comment:
Consider refactoring the MSI suffix into a property in a settings file so that it can be used here and in the MSI project and anywhere it is used.